src/Plugin/DataCollectorPlugin.php

```diff
@@ -24,40 +24,45 @@ class DataCollectorPlugin extends DataCollector implements Plugin
     protected $listenerHandlers = [];
 
     /**
+     * @var array
+     */
+    private $buses = [];
+
+    /**
      * @var Stopwatch
      */
     private $stopwatch;
 
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
     public function __construct(ContainerInterface $container, string $busType)
     {
         $this->stopwatch = new Stopwatch();
-        $this->data['bus_type'] = $busType;
         $this->container = $container;
+        $this->data['bus_type'] = $busType;
         $this->data['messages'] = [];
         $this->data['duration'] = [];
-        $this->buses = [];
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function collect(Request $request, Response $response, \Exception $exception = null)
     {
         foreach ($this->buses as $bus) {
             $busName = $bus->busName();
 
-            $reflClass = new \ReflectionClass($bus);
+            /*$reflClass = new ReflectionClass($bus);
```

Comment:
Should we remove this code block? Non active code is confusing. We could create an issue instead.